hexrd/ui/image_mode_widget.py

```diff
@@ -136,17 +144,12 @@ def enable_image_mode_widget(self, b):
         self.ui.tab_widget.setEnabled(b)
 
     def set_image_mode_widget_tab(self, idx):
-        self.ui.tab_widget.setCurrentIndex(idx)
-
-    def currentChanged(self, index = None):
-        modes = {
-            0: ViewType.raw,
-            1: ViewType.cartesian,
-            2: ViewType.polar,
-            3: ViewType.stereo,
-        }
-        ind = self.ui.tab_widget.currentIndex()
-        self.tab_changed.emit(modes[ind])
+        tab = VIEW_MODE_TO_TAB_INDEX[idx]
+        self.ui.tab_widget.setCurrentIndex(tab)
+
+    def currentChanged(self, index):
+        ind = TAB_INDEX_TO_VIEW_MODE[index]
+        self.tab_changed.emit(ind)
```

Comment:
The `ind` here can also be renamed to `view_mode`.

Reply:
Fixed!